.github/workflows/release-python-runtime.yml

```diff
@@ -27,9 +27,55 @@ jobs:
           R2_ACCOUNT_ID: ${{ secrets.PYODIDE_CAPNP_R2_ACCOUNT_ID }}
           R2_ACCESS_KEY_ID: ${{ secrets.PYODIDE_CAPNP_R2_ACCESS_KEY_ID }}
           R2_SECRET_ACCESS_KEY: ${{ secrets.PYODIDE_CAPNP_R2_SECRET_ACCESS_KEY }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
           bazel build @workerd//src/pyodide:python_bundles @workerd//src/pyodide:bundle_version_info --remote_cache=https://bazel:${{ secrets.BAZEL_CACHE_KEY }}@bazel-remote-cache.devprod.cloudflare.dev
           # boto3 v1.36.0 fails with:
           # NotImplemented error occurred in CreateMultipartUpload operation: Header 'x-amz-checksum-algorithm' with value 'CRC32' not implemented
           pip install 'boto3<1.36.0' requests
           python3 src/pyodide/upload_bundles.py
+
+      - name: Check for open PR and commit changes
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          # Commit changes to python_metadata.bzl and push to branch
+          # Configure git
+          git config --local user.email "action@github.com"
+          git config --local user.name "GitHub Action"
```

Comment:
Nit: Can we give it a more descriptive name?

Reply:
How about release-python-runtime.yml action?

Reply:
I'll try that, hopefully we won't get CLA problems.